webcrypto/subtle_crypto.go

```diff
@@ -246,14 +322,92 @@ func (sc *SubtleCrypto) Sign(algorithm goja.Value, key goja.Value, data []byte)
 // The `signature` parameter should contain the signature to be verified.
 //
 // The `data` parameter should contain the original signed data.
-func (sc *SubtleCrypto) Verify(
-	algorithm goja.Value,
-	key goja.Value,
-	signature []byte,
-	data []byte,
-) *goja.Promise {
-	// TODO: implementation
-	return nil
+//
+//nolint:funlen
+func (sc *SubtleCrypto) Verify(algorithm, key, signature, data goja.Value) *goja.Promise {
+	rt := sc.vu.Runtime()
+	promise, resolve, reject := sc.makeHandledPromise()
+
+	// 2.
+	ab, err := exportArrayBuffer(sc.vu.Runtime(), signature)
+	if err != nil {
+		reject(err)
+		return promise
+	}
+	signatureData := make([]byte, len(ab))
+	copy(signatureData, ab)
```

Comment:
Should we wrap it a function? Like `exportAndCopyArrayBuffer()`

Reply:
I considered it, but I tend to prefer composition over packing everything in a single function 😄 In the Web Platform tests they have this `copyBuffers` helper function everywhere, I could add a similar function in Go `copyBuffers(src, dst goja.ArrayBuffer), what do you think?

Reply:
Considering we only use `exportArrayBuffer` in the context of steps described by the specification as "obtain a copy of the bytes held by X", I modified the function to return a copy of the underlying bytes 👍🏻